src/social-share-button-qr.js

```
@@ -0,0 +1,243 @@
+ */
+
+(function () {
+  /**
+   * Shared bootstrap error helper for the QR extension.
+   * Keeps all console output in one place so it can be easily
+   * swapped for a project-level logger without touching call sites.
+   *
+   * @param {string} message
+   */
+  function _qrWarn(message) {
+    if (typeof console !== "undefined" && typeof console.warn === "function") {
```

ESLint

[error] Unexpected console statement.

(no-console)